spec/Subscriber-spec.ts

```diff
@@ -97,4 +97,15 @@ describe('Subscriber', () => {
     expect(sub1.closed).to.be.false;
     expect(sub2.closed).to.be.true;
   });
+
+  it('should call complete observer without any arguments', () => {
+    const observer = {
+      complete: sinon.spy()
+    };
+
+    const sub1 = new Subscriber(observer);
+    sub1.complete();
+
+    expect(observer.complete.firstCall.args).to.empty;
```

Comment:
Also, I don't know enough about sinon to know if this is how we check this. It seems like a more straight forward test just using a plain object and `expect(arguments.length).to.equal(0)` would do.